node/src/transition_frontier/transition_frontier_actions.rs

```diff
@@ -69,6 +74,9 @@ impl redux::EnablingCondition<crate::State> for TransitionFrontierAction {
                             .map_or(false, |s| s.is_apply_error()),
                     }
             }
+            TransitionFrontierAction::RpcRespondBestTip { .. } => {
+                state.transition_frontier.best_tip().is_some()
```

Comment:
what happens if we don't have a best tip? we just never answer? doesn't feel right. This is a bug in the original code, not introduced in the PR, but just now noticed it.